news/2025/2025-03-12-skin-of-the-year-2024-results.md

```diff
@@ -0,0 +1,93 @@
+
+[![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/phosphor.jpg)](https://osu.ppy.sh/community/forums/topics/1920571)
+
+If it seems like we are going back in time, you might be right. Not only is [Miorine](https://osu.ppy.sh/users/12630408) being mentioned again, but we are once again seeing [Phosphor](https://osu.ppy.sh/community/forums/topics/1920571), a previous contest-winning skin, and they have also earned another badge based on the same skin!
+
+Continuing on with the time travelling theme, Phosphor transports you back to a time when punch cards and floppies were the cutting edge technologies, pairing that with unique gameplay elements and the overall aesthetic, it's no wonder it managed to win the hearts of **787** retro-heads, and get placed at #3.
+
+![](https://assets.ppy.sh/profile-badges/SOTY2024-Phosphor@2x.png)
+
+[![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/fool-moon-night2.0.jpg)](https://osu.ppy.sh/community/forums/topics/1885740)
+
+Yet another remaster has grabbed the hearts and minds of the community, this time around being the [FOOL MOON NIGHT 2.0](https://osu.ppy.sh/community/forums/topics/1885740) skin by [Spoo](https://osu.ppy.sh/users/11805037)!
+
+Spoo took the [original skin](https://osu.ppy.sh/community/forums/topics/1775136)'s idea (inspired by the iconic [FOOL MOON NIGHT](https://osu.ppy.sh/beatmapsets/524026#osu/1141084) beatmap background) and cranked it up to 11, re-drawing each element with much more detail, even incorporating memes, if you know where to look. As a result of their three months of hard work, the community overwhelmingly supported Spoo, bringing in a total of **882** votes and securing the #2 spot!
+
+![](https://assets.ppy.sh/profile-badges/SOTY2024-FOOL-MOON-NIGHT@2x.png)
+
+[![](/wiki/shared/news/2025-03-12-skin-of-the-year-2024-results/freedom-dive-remastered.jpg)](https://osu.ppy.sh/community/forums/topics/2019099)
+
+You could say that 2024 was [Spoo](https://osu.ppy.sh/users/11805037)'s year of the remaster, and it looks like it has paid out it's dividends, with them taking both the #1 and #2 spots!
+
+With yet another throwback, the [original skin](https://osu.ppy.sh/community/forums/topics/1293207) was based on the famous [background](https://images6.alphacoders.com/974/974020.png) used across various FREEDOM DiVE beatmapsets, and it was created all the way back in 2021 for the [first ever skinship skinning contest](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement)! After it received the Spoo™ treatment with a complete re-draw of all elements, it was received with huge praise, so much so that the community deemed it worthy of the #1 spot, with **1078** votes!
```

Comment:
```suggestion
With yet another throwback, the [original skin](https://osu.ppy.sh/community/forums/topics/1293207) was based on the famous [background](https://www.pixiv.net/en/artworks/41256222) used across various FREEDOM DiVE beatmapsets, and it was created all the way back in 2021 for the [first ever skinship skinning contest](https://osu.ppy.sh/home/news/2021-03-08-skinning-contest-announcement)! After it received the Spoo™ treatment with a complete re-draw of all elements, it was received with huge praise, so much so that the community deemed it worthy of the #1 spot, with **1078** votes!
```
Might be better to at least link it to the original artists' page as it is the background used for the Cytus mobile game's inclusion of FREEDOM DiVE.

Reply:
who's the original artist? i can just find various wiki pages from other rhythm games

Reply:
original artist was linked to be [SnowEgg](https://www.pixiv.net/en/users/1755315) who did work on various Rayark background covers like [AXION](https://www.pixiv.net/en/artworks/37924740), [Rainbow Night Sky Highway](https://www.pixiv.net/en/artworks/37992095) and various [DeeMo](https://www.pixiv.net/en/artworks/56764190) / [VOEZ](https://www.pixiv.net/en/artworks/57244675) tracks.

Reply:
i'll just link to https://www.pixiv.net/en/artworks/41256222